implentio/ui-toolkit
# add NavItem component

## Description
This is the navigation item component. Will be used to assemble the navbar

2 Variants:
type = "single" | "group"

## Link to JIRA ticket
[Include the link here.](https://implentio.atlassian.net/jira/software/projects/IUT/boards/10?selectedIssue=IUT-27)

## Testing Instructions
Outline the steps required to test your changes effectively (Replace the following to what makes sense for your PR):
- Fetch and checkout my branch `git fetch && git checkout <feat/IUT-27/NavItem>`
- npm run storybook
- Check to see that the elements are aligned, text is properly showing, and there is a hover effect changing text color to brand-emphasis

## Checklist:
Check what is applicable by putting an 'x' in between the []:
